xilem_web/web_examples/fetch/src/main.rs

```diff
@@ -0,0 +1,181 @@
+// Copyright 2024 the Xilem Authors
+// SPDX-License-Identifier: Apache-2.0
+
+use wasm_bindgen::{JsCast, UnwrapThrowExt};
+use xilem_web::{
+    core::{fork, one_of::Either},
+    document_body,
+    elements::html::*,
+    interfaces::{Element, HtmlDivElement, HtmlImageElement, HtmlLabelElement},
+    memoized_await, App,
+};
+
+use gloo_net::http::Request;
+use serde::{Deserialize, Serialize};
+
+const TOO_MANY_CATS: usize = 8;
+
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Cat {
+    pub url: String,
```

Comment:
We should have the width and the height as well, to avoid the jumping around

Reply:
Oh, I somehow missed, that this is provided by the API, yeah I agree, thanks for the catch.